packages/d2-app/package.json

```diff
@@ -1,18 +1,22 @@
 {
     "name": "@dhis2/cli-app",
+    "bin": {
+        "d2-app": "./bin/d2-app"
+    },
     "version": "0.11.0",
-    "description": "D2 CLI module for interacting with frontend applications",
-    "main": "index.js",
-    "repository": "https://github.com/dhis2/cli",
-    "author": "Austin McGee <austin@dhis2.org>",
+    "main": "src/index.js",
+    "author": "Viktor Varland <viktor@dhis2.org>",
```

Comment:
I see what you did there ;-)

Reply:
Hahaha, oh man, sorry about that.

I did a `rm` of `d2-app`, then regenerated it with the `create cli` tool to allow it to match the new convention as it seemed to predate the create script, and accepted the defaults. :D

Reply:
8352ec4